src/SmartComponents/Systems/SystemsTable.js

```diff
@@ -60,7 +60,23 @@ const SystemsTable = ({
     ]);
 
     const { systemProfile, selectedTags,
-        filter, search, page, perPage, sort } = queryParams;
+        filter: queryParamsFilter, search, page, perPage, sort
+    } = queryParams;
+    const { os: operatingSystemFilter, ...filter } = queryParamsFilter || {};
+    const osFilter = operatingSystemFilter && [{
```

Comment:
Can we extract this into some utility function? The os filter exists in Advisory systems table as well. Probably we need to make same changes there as welll

Reply:
AH! I'll take a look!

Reply:
There we don't need the change, because the table is not querying against the Patch API, but the inventory API and the filter comes directly from there.

Reply:
oh, I'm confused. ok looking more. before I say more. :D